app/views/guests/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1><%= @guest.name %></h1>
+<% @guest.rooms.each do |room| %>
+<div id="<%= room.suite %>">
+<ul>
+  <h3><li>Suite: <%= room.suite %></li></h3>
+  <p>Nightly Rate: $<%= room.rate %></p>
+    <p>Hotel: <%= room.hotel.name %></p>
+</ul>
+</div>
+<% end %>
+<br>
+<div id="add_a_room"> 
+<h3>Add a Room:</h3>
+<%= form_with url: "/guests/#{@guest.id}/guest_rooms", method: :post, local: :true do |f| %>
```

Comment:
I agree with the method and overall setup for this form, I think the URL is not quite RESTful, but it's close... 

When we're writing paths, consider the format of "resource/:which_one". So if we're POST-ing something, that means we're _creating_ something. What thing are we creating here? We're creating a relationship between a Guest and a Room. Do we have a model for that? Yes, it's `GuestRoom`. So, our resource (URL) would be `"guest_rooms"`. Since we're creating this relationship, both of the IDs we need to do it could be contained within the request body, the form data. Since the user is only rqeuired to enter the Room Id, we can make a hidden field for `guest_id` and fill in that value with `@guest.id`. 

Another potential path yo ucould use would be
`url: "/guests/:guest_id/rooms`
and this would still be a POST to the `guest_rooms_controller` `create` action.

Reply:
The reason I would just do `guest_rooms` for the URL is because that's the resource we're creating. 
The alternative of doing `guests/:guest_id/rooms` is because we're adding a specific room to this specific guest. 

I've seen and used both types of paths for this kind of operation and I think both are equally acceptable.